src/components/Menu/Menu.tsx

```diff
@@ -275,7 +275,8 @@ export default function Menu({
             key="professional-requests"
             title={
               <span>
-                Professional requests{" "}
+                Professional reque
+                {allProfessionalPendingRequestsCount > 0 ? "" : "sts"}
```

Comment:
😂

this is kind of amazing. we don't have to support translation or variable menu sizes so sure, why not!

```suggestion
                {allProfessionalPendingRequestsCount > 0 ? "&nbsp;…" : "sts"}
```